### PR TITLE
(SYS-4522) Added new domain for short links

### PR DESCRIPTION
**Pull request by redreceipt**
Currently, the short links that work are something like https://rm2y5.goo.gl/XXXXX which is ugly and so comms will never use it. I added the ability for a better-looking link so they’ll actually want to use short links again.

kapture 2018-11-27 at 10 57 10

Creator
-------

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

Reviewer
--------

- [ ] Run `test` and make sure all tests pass
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@redreceipt @richarddubay Could I have a bit of context on this PR?

**Comment by redreceipt** [comment]
Currently, the short links that work are something like https://rm2y5.goo.gl/XXXXX which is ugly and so comms will never use it. I added the ability for a better looking link so they’ll actually want to use short links again.

**Comment by richarddubay** [comment]
@redreceipt If this is still needed, there are some conflicts that will need to be addressed.

**Comment by redreceipt** [comment]
@richarddubay resolved!

**Comment by IsaacHardy** [comment]
@cdanwards Could you review this PR for @redreceipt ?

File: app.json [hunk 1]
```diff
@@ -46,15 +46,16 @@
           "file": "sentry-expo/upload-sourcemaps",
           "config": {
             "organization": "newspring-church",
-            "project": "app"
+            "project": "app",
+            "authToken": "571cef2c6c50465cae6c87e7cfb1455f7a6edfe2a64a47f9b78d3727bfc17c3b"
```
Comment (cdanwards) [inline app.json]: Should this `authToken` be here? Is this a security vulnerability?

Comment (cdanwards) [reply app.json]: @redreceipt

Comment (redreceipt) [reply app.json]: haha yes that's an issue with `app.json` being in the repo (see #748). I removed that line, should be irrelevant for testing.

**Comment by redreceipt** [comment]
@cdanwards what else do you need?

**Pull request merged**